### ISSUE TITLE
Readium fails when using readium index page as source inside iframe on IE

### ISSUE THREAD

**Issue by mahag** [readium/readium-shared-js#330]
I have an html page with iframe that has source to readium viewer link  as follows

` <iframe width="100%" height="500px"  id="ifjdfnkjvnjjbk"  src="http://example.com/cloud-reader-lite/index.html?epub=epub_content/test.epub"  runat="server"  ></iframe>`

This does open on google chrome but it fails on IE. It shows the book title on top but fails to open the file the loading circle keeps on running and nothing is displayed
![untitled6](https://cloud.githubusercontent.com/assets/10612561/18742404/bbe879bc-80bc-11e6-84d6-b1288ce66429.png)

How could this be solved?

**Comment by rkwright** [comment]
@mahag Are there any errors in the console?

**Comment by mahag** [comment]
Yes there is an access denied error even though I have added document.domain script as suggested when I looked about this issue on some forums.
![untitled1](https://cloud.githubusercontent.com/assets/10612561/18826806/fea0ed5c-83d7-11e6-9951-33886c9e6627.png)

**Comment by mahag** [comment]
I did try few things in order to skip this error. I added correct domain value in my html page and I was able to skip the error at that point of code but I received the same error on another location inside readium JavaScript file as shown in img
![untitled7](https://cloud.githubusercontent.com/assets/10612561/18864129/917047d4-849d-11e6-8e9e-b9966774d3a0.png)
Any ideas?
